.github/workflows/root-ci.yml

```diff
@@ -468,6 +468,7 @@ jobs:
         run: ".github/workflows/root-ci-config/build_root.py
                     --buildtype      ${{ inputs.buildtype }}
                     --platform       ${{ matrix.image }}
+                    --image          registry.cern.ch/root-ci/${{ matrix.image }}:buildready
```

Comment:
Why is this related to improving the instructions?

Reply:
it can be inferred from the `--platform` option. Actually the full script can be simplified...